rhelai-validation/vars.yaml

```diff
@@ -1,5 +1,14 @@
 ---
-# optional
+deploy_rhelai_enabled: true  # Whether to deploy a VM before running tests
+deploy_rhelai_vm_name: rhel-ai  # Name of the VM to create and destroy
+deploy_rhelai_image_name: rhel-ai  # Image to use when creating the VM
+deploy_rhelai_flavor: nvidia  # Flavor to use when creating the VM
+deploy_rhelai_floating_ip: 192.168.122.222  # Floating IP to assign to the VM
+deploy_rhelai_private_key_file: ~/.ssh/rhel-ai.pem  # WARNING Where to (over-)write the ssh key. Must match path in inventory
+deploy_rhelai_pre_cleanup_enabled: true  # Whether to clean up the VM before running tests
+deploy_rhelai_post_cleanup_enabled: false  # Whether to clean up the VM after running tests
+deploy_rhelai_openstack_command: oc rsh openstackclient openstack  # Set to "openstack" if it's available locally
```

Comment:
Probably a nit, but, why do we need this "deploy_rhelai_openstack_command" to be a configuration option ? Perhaps what I am trying to do is to avoid making configuration too complex with things that we can do programmatically.

In ansible we can check if the command "openstack" is available and if not fallback to the "oc rsh openstackclient openstack".

Are these the only two options, right ? Or do we expect people to have other option and therefore we should make it configurable ?

Reply:
It's not quite that simple because the `openstack` command is actually NOT installed by default in the test-container, so this basically serves as a switch to decide whether or not to install it/use it.  (which comes with additional requirements of setting up the correct clouds.yaml)

Ultimately, I considered that we may want all of the `openstack` calls to be replaced with with openstack.cloud ansible collection tasks, but I thought I'd keep it closer to the current code for now and not add a new dependency where it's not really needed.

Reply:
For context, this is how I got here:
1. Call rhoso-rheai deployment code to handle this stuff - worked (with hacks), but not in test-container
2. Copy/paste things from rhoso-rhelai because it's kinda ill-suited anyways - worked, but still not in test-container.
    1. original code uses `oc rsh openstackclient openstack...` (which has lots of advantages IMO)
    2. test-container has easy access to openstack, but not openshift
3. took a quick tour through using openstack.cloud collection - decide it's a medium amount of work and some future risk for little immediate gain
4. Implement this simple string interpolation hack

Reply:
Thanks for the explanation, I don't think we need to make it a boolean. It's OK as-is given the situation. Also, these playbooks are intended to test/CI so I think we could remove/change configuration variables in the future without having to go thru the whole deprecation period etc...

Reply:
From my point of view, it is okay to leave it as it is right now if we track somewhere that we later want to remove the `deploy_rhelai_openstack_command` variable and replace it with `openstack.cloud`.

This concern is partially related to this comment [1]. 

[1] https://github.com/openstack-k8s-operators/rhelai-validation/pull/2#discussion_r1973157541